### DISCUSSION TITLE
Animation frame rate decrease

### DISCUSSION THREAD

**Discussion by YangSpring114** [AvaloniaUI/Avalonia#11489] [Q&A] [answered]
After packaging my project using the release feature of VS2022, the animation frame rate of the program significantly decreased, but it was normal for me to run in debugging mode. Is this my problem or a bug?

This is the frame rate details in debugging mode:

![DX BA5U1`8J)JU)XMQM6)39](https://github.com/AvaloniaUI/Avalonia/assets/99802662/65d2d6de-c8fb-41cc-9bc9-1a274ee67f48)

This is the frame rate details when the program runs after packaging:

![3G9XZ2 @JKIDF0}%RCD)94G](https://github.com/AvaloniaUI/Avalonia/assets/99802662/e7442f87-e0c3-4454-8685-0f8d530212bf)

**Comment by maxkatz6** [comment, marked as answer]
What Avalonia version are you using?
Note, that before 0.10.19 with .NET 6/7 you had to publish application with "win7-x64" or "win7-x86" runtime identifier to have GPU support.
After 0.10.19 no extra steps are needed.

**Reply by YangSpring114** [reply]
The Avalonia version I am using is 11.0.0 preview4 and the operating system is Windows 10

**Reply by maxkatz6** [reply]
> 11.0.0 preview4

That's too old. Please upgrade.
Also, preview4 has the same issue with runtime identifier - you need to specify win7 version there ("win7-x64" or "win7-x86", win10 also works). Or upgrade.

**Reply by YangSpring114** [reply]
Alright, the frame rate of the animation has been increased now. Thank you for your help!
